docs/api/cmd.rst

```diff
@@ -3,3 +3,20 @@ cmd2.Cmd
 
 .. autoclass:: cmd2.cmd2.Cmd
     :members:
+
+    .. attribute:: help_error
```

Comment:
Ok, so this is the way to document instance attributes then?  It looks pretty reasonable, just have to remember to do it is the trick.

Reply:
Yup, this is option #3 from #821. We don't pull any attribute documentation from the source code, so it has to be created manually in the documentation. This is the way.